packages/web3-core-helpers/src/errors.js

```diff
@@ -76,5 +76,29 @@ module.exports = {
     },
     TransactionOutOfGasError: function(receipt) {
         return this.TransactionError('Transaction ran out of gas. Please provide more gas:\n' + JSON.stringify(receipt, null, 2), receipt);
+    },
+    ContractMissingABIError: function() {
```

Comment:
Is it possible to have those error messages covered with real unit tests? What I mean with this would be to have those functions tested as a unit and also tested in the unit/integration mix tests we have in the contract tests of the legacy codebase.

Reply:
> Is it possible to have those error messages covered with real unit tests?

If the error tests include the entire message I think we get good test coverage for this without needing to duplicate anything. 

Another thing is that if you look at the coverage report for helpers you can see the some errors are *not* being thrown in the tests - that information is useful but would be hidden by direct units.

Reply:
> If the error tests include the entire message I think we get good test coverage for this without needing to duplicate anything.

Green coverage !== good unit tests. I would test them as a unit as the definition of unit tests are.

Reply:
Have looked at this again... I think we just disagree. Hitting these errors via the methods which call them seems like the correct approach to me.